src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -453,7 +453,7 @@ public static class Features
                 public static readonly string GitShallowDepth = "agent.source.git.shallowFetchDepth";
                 public static readonly string SkipSyncSource = "agent.source.skip";
                 public static readonly string EnableAdditionalMaskingRegexes = "agent.enableadditionalmaskingregexes";
-                public static readonly string UseMaskingPerformanceEnhancements = "agent.agentUseMaskingPerformanceEnhancements";
+                public static readonly string UseMaskingPerformanceEnhancements = "agent.useMaskingPerformanceEnhancements";
```

Comment:
>UseMaskingPerformanceEnhancements [](http://example.com/codeflow?start=46&length=33)

Why aren't we converging on the term below? Now, for sure the new masker is intended to provide perf improvements. It's also a delivery mechanism for new secret detections, though. 

`agent.enableNewSecretMasker`.

EnableNewSecretMasker

Reply:
Yes, I discuss renaming this here: https://github.com/microsoft/azure-pipelines-agent/pull/5154#discussion_r2003760457

Reply:
Resolving and let's discuss there.